### DISCUSSION TITLE
年会（起） | /posts/annual-meeting/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#603] [Announcements]
https://yuanfan.rbind.io/posts/annual-meeting/

**Comment by meeeeeeeeo** [comment]
今年年会我也参与排节目了，也是背景板跳舞，不过节目组织者是隔壁组的熟人，关系还可以，就帮忙撑场面了。。。
我们节目的主题还是你的上个版本，带一点“我爱上班”，背景歌声还有啥啥啥为公司奋斗的，然后排练中途公司还来了波降本增效，扬言要砍各种活动预算，我心里吐槽个不停，“钱都不给谁给你奋斗”

**Reply by earfanfan** [reply]
哈哈，我有点好奇等到你们表演那天，背景板们的演出服会是撒，当然我也好奇等到那一天我们会穿撒……

**Reply by meeeeeeeeo** [reply]
俺们年会已经过去一个月了，排练加别的分会场场务，人都累趴了 :disappointed:
演出服是公司统一发的卫衣加小丑头套加墨镜，还被抓拍了照片 :joy:

**Reply by earfanfan** [reply]
哦，原来你们的年会是按公历新年算的啊，我们公司通常是按农历新年算，在春节放假前办，听说安排在下个月1号。完了，听你一提我才想起来到时候表演节目有可能会被录像，俺顿感，苍天啊……

又听说我们那个名为相亲相爱的歌舞节目会是开场，大家都觉得第一个上表演完就可以下去吃饭了，还蛮好的，哈哈。话又说回来，如果你今年干得好的话，其实有可能明年年会还会喊（使）上（唤）你的。

**Reply by meeeeeeeeo** [reply]
巧了我们节目也是开场，确实跳完了就看别人表演了，不过你们怎么是别人在下面吃你们在上面跳
抓拍指的是正常多方位多角度录视频之外，被节目组拍下了很“亮眼”的照片

**Reply by earfanfan** [reply]
我们公司年会形式就是吃饭，所以就会是下面的人吃饭，上台表演节目的在台上表演。嗯，俺知道抓拍就是指正常的抓拍，哈哈，但是每个普通人都顶不住各种角度地拍啊，大家都不太会表情管理，我已经能想象到时候吃席群众们一边放大照片或视频一边吐槽的场景了。
